src/environments/environment.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
   production: false,
-  baseUrl :'http://34.143.225.1/template/api/v1/'
-  // baseUrl :'http://localhost:5000/api/v1/'
+  // baseUrl :'http://34.143.225.1/template/api/v1/'
+  baseUrl :'http://localhost:5000/template/api/v1/'
```

Comment:
We need to shift this property to env.js file and wherever we're using this, we need to make changes like how we did for customAuth

Reply:
Okay